### PR TITLE
.github: Add workflows to build conda/libtorch images

### PR DESCRIPTION
**Pull request by seemethere**
Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

### CODE REVIEW

File: common/install_magma.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 set -eou pipefail
 
+# Check https://anaconda.org/pytorch/magma-cuda${cuda_version_nodot} to see if there's a magma version available
```
Comment (walterddr) [inline common/install_magma.sh:5]: is this an instruction to developer when changing the MAGMA_VERSION below?

**Review by walterddr — APPROVED** [review]

File: .github/workflows/build-conda-images.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      matrix:
+        cuda_version: ["10.2", "11.1", "11.3", "cpu"]
+    env:
+      CUDA_VERSION: ${{ matrix.cuda_version }}
+    steps:
+      - name: Checkout PyTorch
+        uses: actions/checkout@v2
+      - name: Build Docker Image
+        run: |
+          conda/build_docker.sh
```
Comment (janeyx99) [inline .github/workflows/build-conda-images.yml:26]: So these images get built when there are changes within the builder repo, right? This wouldn't be replacing the circle docker jobs just yet?

**Review by janeyx99 — APPROVED** [review]

**Review by malfet — APPROVED** [review]

File: common/install_magma.sh (hunk 1 above)
Comment (seemethere) [reply common/install_magma.sh:5]: This is just a comment to trigger the workflows, I'll probably drop this commit once I verify that everything's run

File: .github/workflows/build-conda-images.yml (hunk 2 above)
Comment (seemethere) [reply .github/workflows/build-conda-images.yml:26]: Yes these are only for the docker images we currently use to build the binary builds

**Pull request merged**